brigade-worker/src/index.ts

```diff
@@ -8,7 +8,8 @@
  * - `BRIGADE_EVENT_PROVIDER`: The name of the event provider, such as `github` or `dockerhub`
  * - `BRIGADE_PROJECT_ID`: The project ID. This is used to load the Project
  *   object from configuration.
- * - `BRIGADE_COMMIT`: The VCS commit ID (e.g. the Git commit)
+ * - `BRIGADE_COMMIT_ID`: The VCS commit ID (e.g. the Git commit)
+ * - `BRIGADE_COMMIT_REF`: The VCS full reference
```

Comment:
Adding something like "This defaults to `refs/head/master`" would be even more helpful imho.
So that we can do understand that the value for BRIGADE_COMMIT_ID can be unset but BRIGADE_COMMIT_REF.